example/pubspec.yaml

```diff
@@ -11,6 +11,9 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+#  advanced_storybook: ^1.0.2
+  flutterbook: ^0.1.6
```

Comment:
I really like the idea with a storybook!

However, I have some concerns about this particular lib. It seems to be not really popular and well-maintained. We should be careful about what we depend on -- how about we pick something that's reached a stable version, and has a community around it, such as https://github.com/widgetbook/widgetbook ?